### PR TITLE
added new category to integration validations

### PR DESCRIPTION
**Pull request by orhovy**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
reference: https://github.com/demisto/etc/issues/31038

## Description
Added the new category

## Screenshots
Paste here any images that will help the reviewer

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by bakatzir — COMMENTED** [review]
let's add a test case the approves it: https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/common/tests/integration_test.py#L396
and rn

File: demisto_sdk/commands/common/constants.py [hunk 1]
```diff
@@ -781,7 +781,7 @@ class PB_Status:
 INTEGRATION_CATEGORIES = ['Analytics & SIEM', 'Utilities', 'Messaging', 'Endpoint', 'Network Security',
                           'Vulnerability Management', 'Case Management', 'Forensics & Malware Analysis',
                           'IT Services', 'Data Enrichment & Threat Intelligence', 'Authentication', 'Database',
-                          'Deception', 'Email Gateway', 'Identity and Access Management']
+                          'Deception', 'Email Gateway', 'Identity and Access Management', 'File Integrity Management']
```
Comment (roysagi) [inline demisto_sdk/commands/common/constants.py:784]: This will this let `init` create this integration type right?

Comment (orhovy) [reply demisto_sdk/commands/common/constants.py:784]: right

**Review by bakatzir — APPROVED** [review]
great. please just add a line in the CHANGELOG.

**Pull request merged**